src/dataAssimilation/DataAssimilation.cpp

```diff
@@ -0,0 +1,116 @@
+/// \file      DataAssimilation.cpp
+/// \brief
+/// \date      May 18, 2021
+/// \author    My Linh Wuerzburger
+/// \copyright <2015-2021> Forschungszentrum Juelich. All rights reserved
+
+#include "DataAssimilation.h"
+#include "../domain/DomainData.h"
+#include "../TCP/TCPServer.h"
+#include "mpi.h"
+#include "TemperatureSourceChanger.h"
+
+DataAssimilation::DataAssimilation(const SolverController &solver_controller,
+                                   FieldController *field_controller,
+                                   const Settings::Settings &settings) :
+        m_settings(settings),
+        m_field_controller(field_controller),
+        m_solver_controller(solver_controller),
+        m_new_field_u(Field(FieldType::U)),
+        m_new_field_v(Field(FieldType::V)),
+        m_new_field_w(Field(FieldType::W)),
+        m_new_field_p(Field(FieldType::P)),
+        m_new_field_T(Field(FieldType::T)),
+        m_new_field_C(Field(FieldType::RHO)) {
+    m_logger = Utility::create_logger(typeid(this).name());
```

Comment:
this line can be a member initializer lists

Reply:
475b423